tests/integration/targets/win_listen_ports_facts/aliases

```diff
@@ -0,0 +1 @@
+windows/adds/group1
```

Comment:
This isn't a valid group, please use

```suggestion
shippable/windows/group5
```

Are you also able to let me know where you got this value? I've seen a few PRs with this type of format and am wondering if our docs are out of date.

Reply:
Group updated. I got it from other modules, as I couldn't find in the docs where this value was from.